packages/model-viewer/src/features/scene-graph/material.ts

```diff
@@ -65,6 +66,18 @@ export class Material extends ThreeDOMElement implements MaterialInterface {
       gltfMaterial.emissiveFactor = [0, 0, 0];
     }
 
+    if (gltfMaterial.doubleSided == null) {
+      gltfMaterial.doubleSided = false;
+    }
+
+    if (gltfMaterial.alphaMode == null) {
+      gltfMaterial.alphaMode = 'OPAQUE';
+    }
+
+    if (gltfMaterial.alphaCutoff == null) {
+      gltfMaterial.alphaCutoff = 0.5;
+    }
```

Comment:
@timmmeh I should have noticed this in code review; we fill in all the defaults at load time, which simplifies logic down the road. I may end up creating a new set of types to represent this, as I'm tired of all the `!`s.